supervisord.conf

```diff
@@ -4,7 +4,7 @@ username = dummy
 password = dummy
 
 [supervisord]
-logfile = /srv/logs/supervisord.log
+logfile = /var/log/supervisor/supervisord.log
```

Comment:
We want to move all logs to `/srv/logs`, not do the opposite thing. See https://confluence.percona.com/x/TIeUAg

Reply:
Process control system logs aren't mentioned in the document. This fact was [pointed out](https://jira.percona.com/browse/PMM-4561?focusedCommentId=240937&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-240937) by our QA team.

Reply:
It is mentioned now. I also added a comment to the ticket.

Please revert that change.

Reply:
@hors we should also use `/srv/logs/supervisord.log` in Ansible playbooks.

Reply:
Cross-reference to https://github.com/Percona-Lab/percona-images/pull/36#discussion_r319905466

Reply:
supervisord.log location is reverted.